code/datums/id_trim/jobs.dm

```diff
@@ -831,19 +831,10 @@
 	subdepartment_color = COLOR_MEDICAL_BLUE
 	sechud_icon_state = SECHUD_PARAMEDIC
 	minimal_access = list(
-		ACCESS_BIT_DEN,
-		ACCESS_CARGO,
-		ACCESS_CONSTRUCTION,
-		ACCESS_HYDROPONICS,
 		ACCESS_MAINT_TUNNELS,
 		ACCESS_MECH_MEDICAL,
 		ACCESS_MEDICAL,
-		ACCESS_MINERAL_STOREROOM,
-		ACCESS_MINING,
-		ACCESS_MINING_STATION,
```

Comment:
i think exceptions could be made here along with at least EVA access too?

Reply:
We have public mining shuttles to handle issues with paramedics heading out to save miners, and paramedics both have access to a MODsuit, but do not have EVA access on live.

Maybe there is a problem for paramedics looking to get to the lowest level of icebox, but that area is significantly more attached to the station than lavaland mining.

Reply:
You mean the segment between the EVA hallway and the lava bridge?

That's not nearly as common as just dying at that door on the bridge itself, or on the wastes themselves, in my experience. Which is accessible via the public access.